### PR TITLE
fix(version): include main as default production branch

### PR DESCRIPTION
**Pull request by corveen**
New Git repositories do not have **master** branch, instead they have **main** branch.

Condo assigns "production" build quality only to builds from master branch. This results in builds from main branch, being _alpha_ builds.

This PR addresses this issue. Backward compatibility is maintained - master branch builds will remain as before.

### CODE REVIEW

**Comment by patrickserrano** [comment]
@sjk07 I know the docs are mainly placeholders, but should this change be documented somewhere so it's obvious condo will handle `main` and `master` appropriately without configuration in `condo.build`?

**Review by dmccaffery — DISMISSED** [review]
thanks for contributing; LGTM!

**Comment by dmccaffery** [comment]
Ah; I see @patrickserrano and @sjk07 are on this (I reacted to notification on mobile and missed this) -- they are more appropriate approvers these days. Apologies!

**Comment by sjk07** [comment]
@corveen Thanks for the contribution! this looks great! Would you mind following the commit conventions before merging this ex. `fix(version): include main as default production branch`? other than that LGTM!

@patrickserrano We should start with documentation on the default cycle, then we can expand the pre-built selectable cycles and finally custom modification. This can probably be a new issue since I don't think this exists atm.

@dmccaffery no worries, you are always welcome and afaik still a maintainer 😉

**Review by sjk07 — APPROVED** [review]

**Pull request merged**
